q2_moshpit/busco/busco.py

```diff
@@ -40,69 +123,138 @@ def evaluate_busco(
+        busco_results
+    )
 
-        # Zip graphs for user download
-        # Result included in final output (file for download)
-        zip_name = os.path.join(output_dir, "busco_plots.zip")
-        q2_moshpit.busco.utils._zip_busco_plots(
-            paths_to_plots=paths_to_plots,
-            zip_path=zip_name
-        )
+    vega_json = json.dumps(context)
+    vega_json_summary = json.dumps(marker_summary_spec)
+    vega_json_summary_selectable = json.dumps(selectable_summary_spec)
+
+    # Copy BUSCO results from tmp dir to output_dir
+    TEMPLATES = os.path.join(
+        os.path.dirname(os.path.dirname(__file__)), "assets"
+    )
+    index = os.path.join(TEMPLATES, "busco", "index.html")
+    details = os.path.join(TEMPLATES, "busco", "detailed_view.html")
+    table = os.path.join(TEMPLATES, "busco", "table.html")
+    copytree(
+        src=os.path.join(TEMPLATES, "busco"),
+        dst=output_dir,
+        dirs_exist_ok=True
+    )
+
+    table_json = _get_feature_table(busco_results)
+    stats_json = _calculate_summary_stats(busco_results)
+
+    # Render
+    tabbed_context = {
+        "tabs": [
+            {"title": "QC overview", "url": "index.html"},
+            {"title": "Sample details", "url": "detailed_view.html"},
+            {"title": "Feature details", "url": "table.html"}
+        ],
+        "vega_json": vega_json,
+        "vega_summary_json": vega_json_summary,
+        "vega_summary_selectable_json": vega_json_summary_selectable,
+        "table": table_json,
+        "summary_stats_json": stats_json,
+        "page_size": 100
+    }
+    templates = [index, details, table]
+    q2templates.render(templates, output_dir, context=tabbed_context)
+
+    # Final cleanup, needed until we fully migrate to Bootstrap 5
+    _cleanup_bootstrap(output_dir)
+
+
+def evaluate_busco(
+    ctx,
+    bins,
+    mode="genome",
+    lineage_dataset=None,
+    augustus=False,
+    augustus_parameters=None,
+    augustus_species=None,
+    auto_lineage=False,
+    auto_lineage_euk=False,
+    auto_lineage_prok=False,
+    cpu=1,
+    config=None,
+    contig_break=10,
+    evalue=1e-03,
+    force=False,
+    limit=3,
+    long=False,
+    metaeuk_parameters=None,
+    metaeuk_rerun_parameters=None,
+    miniprot=False,
+    scaffold_composition=False,
+    num_partitions=None
+):
+
+    kwargs = {
+        k: v for k, v in locals().items()
+        if k not in ["bins", "ctx", "num_partitions"]
+    }
+
+    _evaluate_busco = ctx.get_action("moshpit", "_evaluate_busco")
+    partition_mags = ctx.get_action("moshpit", "partition_sample_data_mags")
+    collate_busco_results = ctx.get_action("moshpit", "collate_busco_results")
+    _visualize_busco = ctx.get_action("moshpit", "_visualize_busco")
+
+    (partitioned_mags, ) = partition_mags(bins, num_partitions)
+    results = []
+    for mag in partitioned_mags.values():
+        (busco_result, ) = _evaluate_busco(mag, **kwargs)
+        results.append(busco_result)
```

Comment:
This change would be in tandem with the change proposed in the `_evaluate_busco` function.

```suggestion
    kwargs = {
        k: v for k, v in locals().items()
        if k not in ["bins", "ctx", "num_partitions"]
    }
    
    common_args = _process_common_input_params(
        processing_func=_parse_busco_params, params=kwargs
    )

    _evaluate_busco = ctx.get_action("moshpit", "_evaluate_busco")
    partition_mags = ctx.get_action("moshpit", "partition_sample_data_mags")
    collate_busco_results = ctx.get_action("moshpit", "collate_busco_results")
    _visualize_busco = ctx.get_action("moshpit", "_visualize_busco")

    (partitioned_mags, ) = partition_mags(bins, num_partitions)
    results = []
    for mag in partitioned_mags.values():
        (busco_result, ) = _evaluate_busco(mag, common_args)
        results.append(busco_result)
```